### DISCUSSION TITLE
Multiple exec-once Not Working

### DISCUSSION THREAD

**Discussion by jrock2004** [hyprwm/Hyprland#1164] [Q&A]
Before opening a bug I want to make sure I am doing this right. I have tried a few ways. Waybar opens just fine but the other too when I go and do a `ps aux` I am not seeing those apps running. Am I doing something wrong? Maybe there is a log file I can look at to see if those other 2 error out?

Attempt 1:
```
exec-once   = waybar &
exec-once   = hyprpaper &
exec-once   = idle.sh 4 &
```

Attempt: 2:
```
exec-once   = waybar & hyprpaper & idle.sh 4 &
```

I checked that those 2 other apps are in my path

**Comment by jrock2004** [comment]
Ok, so hyprpaper was crashing so I switched to swaybg and that now works. The shell script, I had to just put an absolute url to that shell script and now it works

**Reply by vaxerski** [reply]
typing `idle.sh` in a terminal is not valid so it wouldn't be launched.

**Reply by AndyP3r3z** [reply]
@vaxerski But he said `idle.sh` is in his `$PATH`, why shouldn't it work? ¯⁠\⁠_⁠(⁠ ͠⁠°⁠ ͟⁠ʖ⁠ ⁠°͠⁠ ⁠)⁠_⁠/⁠¯
